docs/user/trusted-publishers/security-model.md

```diff
@@ -4,23 +4,48 @@ title: Security Model and Considerations
 
 # Security model and considerations
 
-## General considerations
-
-While more secure than passwords and long-lived API tokens, OIDC publishing
-is not a panacea. In particular:
+Trusted Publishing is primarily designed to be a more secure alternative to
+the long-lived API tokens that have traditionally been used for publishing to
+PyPI.
 
-* Short-lived API tokens are still sensitive material, and should not be
-  disclosed (ideally not at all, but certainly not before they expire).
+In recent years, theft of credentials such as API tokens has [played a major
+role in cyber attacks]. The reason for this is the unfortunate reality that
+managing credentials can be unexpectedly difficult. Trusted Publishing addresses
+this risk by using short-lived tokens instead of long-lived tokens that can
+easily get misplaced, leaked in logs, stolen by malware, or any number of other
+ways.
```

Comment:
I think the key distinction we should make here is that long-lived tokens need to be stored and do not quickly expire. That somewhat reduces the likelihood that they will "misplaced, leaked in logs, stolen by malware" (although it's still possible) but _also_ reduces the impact if they are compromised.

Reply:
Good point, let me know what you think of the new version I just pushed 👍